frontend/lib/swr-cache/adapters/index.ts

```diff
@@ -11,10 +11,7 @@ export const getCache = () => {
    const client = new Redis(REDIS_URL, {
       connectTimeout: 500,
       // Retry, connect every once in a while as cache misses / failures are OK
-      retryStrategy: (times) => {
-         times = Math.min(20, times);
-         return Math.min(50 + 2 ** (times + 3), 2 * 60 * 1000);
-      },
+      retryStrategy: (times) => Math.min(50 + 2 ** (times + 3), 2 * 60 * 1000),
```

Comment:
I did the double-min because I figured if `times` got into the hundreds / thousands it might crash.

But I was wrong... looks like it just results in `Infinity`:

![image](https://user-images.githubusercontent.com/26855/217399606-ddc52e63-b430-4c72-b712-16d9a6593cc4.png)

Which seems to work fine with `Math.min()`

Reply:
I had that "oh THAT'S why he did that!" thought immediately after committing it and checked it, but luckily it worked anyway.